Monika After Story/game/script-topics.rpy

```diff
@@ -10060,12 +10162,21 @@ label monika_beingevil:
                     $ persistent._mas_pm_monika_evil_but_ok = True
                     $ mas_gainAffection()
 
-                    m 1rksdla "[player]..."
-                    m 1eka "That means a lot..."
-                    m 1eksdlc "I know that I've done some bad things in the past..."
-                    m 3eka "But I've learned from them and grown as a person since."
-                    m 1eka "I'm really glad you're willing to forgive me, [player]."
-                    m 1hub "I promise I'll be the best I can be, just for you!"
+                    if mas_isMoniNormal(higher=True):
+                        m 1rksdla "[player]..."
+                        m 1eka "That means a lot..."
+                        m 1eksdlc "I know that I've done some bad things in the past..."
+                        m 3eka "But I've learned from them and grown as a person since."
+                        m 1eka "I'm really glad you're willing to forgive me, [player]."
+                        m 1hub "I promise I'll be the best I can be, just for you!"
+                    else:
+                        m 2rsd "Would be nice to see some then."
```

Comment:
I don't understand this line, would she like to see some evil? Would she like to see you being okay with it? Maybe I'm just not reading it correctly.

Reply:
She'd like to see the forgiveness you say she deserves. This is low affection path, so presumably you haven't shown her much of it.

Reply:
Oh I see. Though I think after being called evil, she would be more careful in sounding harsh, if she's feeling insecure about that topic already? I get that it's supposed to be a random moment of weakness that "came out of nowhere", but still.

How about a line like this instead?:

"m 2rsd "Really? It sure doesn't feel like it these days..."

Or something along those lines. I think it feels less like she's attacking the player, and more that she's missing the forgiveness part in your interactions. Just my two cents, I don't really mind either way since I'm not going down there anytime soon.